app/src/main/java/net/gsantner/markor/activity/DocumentEditFragment.java

```diff
@@ -663,15 +663,14 @@ public String getFragmentTag() {
     // Save the file
     // Only supports java.io.File. TODO: Android Content
     public boolean saveDocument(final boolean forceSaveEmpty) {
-        if (!isAdded()) {
+        if (!isAdded() || _hlEditor == null || _document == null) {
```

Comment:
I've never seen this fail on my phone, but I noticed it occasionally failing in the emulator, so I added some defensive code.